xcodeproj/internal/xcodeprojinfo.bzl

```diff
@@ -71,6 +71,7 @@ _BUILD_TEST_RULES = {
     "macos_build_test": None,
     "tvos_build_test": None,
     "watchos_build_test": None,
+    "test_suite": None,
```

Comment:
nit: this should only be the `*_build_test` rules.

Reply:
👍 will put this in a separate constant